### PR TITLE
1. Disabling clicking on labels and far-right whitespace to toggle check...

### PR DESCRIPTION
**Pull request by eshon**
Fix provided:

1. Disabling clicking on labels and far-right whitespace to toggle checkboxes
2. Removing closing input tags because input tags do not have closing tags 

For https://jira.codehaus.org/browse/GEOS-6948

### CODE REVIEW

**Comment by jodygarnett** [comment]
Hi Eva! Is there a jira issue or something for this one? I am familiar (and frustrated by this issue).

File: src/web/core/src/main/java/org/geoserver/web/services/BaseServiceAdminPage.java [hunk 1]
```diff
@@ -55,16 +55,16 @@
  *   &lt;li>
```
Comment (jodygarnett) [inline src/web/core/src/main/java/org/geoserver/web/services/BaseServiceAdminPage.java]: Update header to say (c) 2014-2015

File: src/web/core/src/main/java/org/geoserver/web/services/BaseServiceAdminPage.java [hunk 2]
```diff
@@ -55,16 +55,16 @@
...
  *       &lt;p class="instruct">
  *       &lt;/p>
  *     &lt;/li>
- *   
+ *
  * </wicket:extend>
  *   </pre>
  * </p>
- * 
+ *
```
Comment (jodygarnett) [inline src/web/core/src/main/java/org/geoserver/web/services/BaseServiceAdminPage.java]: lots of small white space changes

**Comment by jodygarnett** [comment]
So this looks good to me, I would like to manually merge to master (so I can clean up the headers) and then backport to stable. For reference Eva is covered under boundless contribution agreement.

Anyone else want to review this one?

**Comment by aaime** [comment]
Jody, any chance all the whitespace changes can be removed as well?

**Comment by aaime** [comment]
Btw, just to clarify, I can see the whitespace changes are improvements too (removal of extra space at endlines and so on), it's just that in order to backport to the stable series a line by line check would be a good thing to do, and like this the change is massive.

**Comment by aaime** [comment]
Jody, maybe during your manual merge you can do that line by line check? ;-)

File: src/web/core/src/main/java/org/geoserver/web/data/layer/CascadedWFSStoredQueryAbstractPage.html [hunk 3]
```diff
@@ -18,7 +18,7 @@
 
 <wicket:fragment wicket:id="newPage">
   <div>
-    <wicket:message key="chooseNativeName">Layer name </wicket:message> <input type="text" wicket:id="nativeName"></input>
+    <wicket:message key="chooseNativeName">Layer name </wicket:message> <input type="text" wicket:id="nativeName">
```
Comment (jodygarnett) [inline src/web/core/src/main/java/org/geoserver/web/data/layer/CascadedWFSStoredQueryAbstractPage.html]: Why are we removing the training </input> ?

Should this be:

````
<input type="text" wicket:id="nativeName"/>
````

Comment (aaime) [inline src/web/core/src/main/java/org/geoserver/web/data/layer/CascadedWFSStoredQueryAbstractPage.html]: From what I read <input> is valid in HTML, if you want valid XHTML is should be <input/> instead.
I'm not sure, but believe that some of our CSS was depending on the browser not to enter in quircks mode, and wondering if using <input> alone would do that (sorry, little web frontend experience...)

Comment (ohmu808) [inline src/web/core/src/main/java/org/geoserver/web/data/layer/CascadedWFSStoredQueryAbstractPage.html]: Not clear on what you mean by "training".
Yes, it should be <input /> but was just trying to focus on getting this ticket done and most browsers are ok with just <input>.

Comment (jodygarnett) [inline src/web/core/src/main/java/org/geoserver/web/data/layer/CascadedWFSStoredQueryAbstractPage.html]: Training was a typo, updated my comment above. I am going through and removing whitespace locally.

Only one that has given me pause is:
```
<input type="radio">mode 1</input>
```

Being turned into:
```
<input type="radio">mode 1
```

**Comment by jodygarnett** [comment]
Replaced by #996

**Pull request closed without merging**